CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## Unreleased
+FEATURES:
```

Comment:
Would you also mind updating the documentation to include the accessor as a attribute? Similar to something like https://registry.terraform.io/providers/hashicorp/vault/latest/docs/resources/github_auth_backend#attributes-reference. Thanks!

Reply:
Ah, my bad -- will update now